.github/workflows/test-wheel-windows.yml

```diff
@@ -84,8 +84,11 @@ jobs:
         continue-on-error: true
 
       - name: Update driver
+        env:
+          DRIVER_MODE: ${{ matrix.DRIVER_MODE }}
+          GPU_TYPE: ${{ matrix.GPU }}
         run: |
-          .github/workflows/install_gpu_driver.ps1
+          ci/tools/install_gpu_driver.ps1
 
       - name: Ensure GPU is working
```

Comment:
After `nvidia-smi` is called, add a check to run it again, grep the output, and make sure it contains `${{ matrix.DRIVER_MODE }}` (WDDM, TCC, or MCDM) as expected.